tools/flake8_plugin.py

```diff
@@ -45,13 +47,27 @@ def visit_ImportFrom(self, node: ast.ImportFrom) -> None:
                 and any(x.name in {"force_bytes", "force_str"} for x in node.names)
             ):
                 self.errors.append((node.lineno, node.col_offset, S006_msg))
+            elif (
+                "tests/" not in self.filename
+                and "sentry/testutils/" not in self.filename
+                and "sentry/utils/pytest" not in self.filename
```

Comment:
we should move this into `testutils`

Reply:
I don't disagree, but that is a change I would like to do separate of these changes.

Reply:
let's do that first then 👍

Reply:
Ok. Moving those pytest helpers isn't going to be a trivial fix. It is going to take a series of pull requests but I'll tag you as a reviewer on all of them.

Reply:
I think we're all done with moving utils.pytest into testutils. Anything else to address before I can get this merged?

Reply:
I'd like to see the silo testutils stuff moved into `testutils` since that's what caused the incident in the first place

Reply:
I've moved the decorate_unit_tests module to be under testutils as well.